quantum/process_keycode/process_leader.c

```diff
@@ -35,31 +35,40 @@ uint16_t leader_time = 0;
 uint16_t leader_sequence[5] = {0, 0, 0, 0, 0};
 uint8_t leader_sequence_size = 0;
 
+void qk_leader_start(void) {
```

Comment:
Since this can be called from a user keymap now, shouldn't this have `if (!leading) ` wrapping the function block?

Reply:
Ah, yes, true.  Then could almost take out that check in the `process_leader` function. 

But this should have this, for sure: 
```suggestion
 void qk_leader_start(void) {
   if (leading) { return; ]
```